components/homme/src/share/gllfvremap_mod.F90

```diff
@@ -184,10 +184,10 @@ subroutine gfr_init(par, elem, nphys, check, boost_pg1)
           ! Document state of pg1. dcmip2016_test1 shows it is too coarse. For
           ! boost_pg1 = true, stepon's DSS loop needs to be separated from its
           ! tendency application loop.
-          write(iulog,*), 'gfr> Warning: pg1 is too coarse; see comments at top of gllfvremap_mod.F90'
+          write(iulog,*) 'gfr> Warning: pg1 is too coarse; see comments at top of gllfvremap_mod.F90'
           if (.not. gfr%boost_pg1) then
-             write(iulog,*), 'gfr> Warning: If you want to try pg1, use the boosted-accuracy &
-                  boost_pg1 option and call gfr_pg1_reconstruct(_topo).'
+             write(iulog,*) 'gfr> Warning: If you want to try pg1, use the boosted-accuracy &
+                  &boost_pg1 option and call gfr_pg1_reconstruct(_topo).'
```

Comment:
Would omitting & on second line work?

Reply:
It works with GNU compiler, but not XL.
According to the link below, "Lines can be broken within a string constant, in which case the initiating ampersand is mandatory"
http://fortranwiki.org/fortran/show/Continuation+lines